expression/builtin_time.go

```diff
@@ -5556,6 +5557,16 @@ func (b *builtinAddStringAndStringSig) evalString(row chunk.Row) (result string,
 		}
 		return "", true, err
 	}
+
+	check := arg1Str
```

Comment:
I think it would be useful to add a comment here about why this is needed

Reply:
Yeah, In fact I think add the code in func `isDuration` or create a new func `maybeErrDuration` may be butter, but `isDuration` is a util func, I'm afraid to get new wrong, so I add the code here. I may add some cmment or create a new func, but which file may i add the func?

Reply:
I will check the other concor case and try to solve them